src/example-types/metricGroups/fieldValuesGroupStats.js

```diff
@@ -9,9 +10,22 @@ let getSortField = field => {
   return field
 }
 
-let drilldown = ({ field, drilldown }, schema) => ({
-  term: { [getField(schema, field)]: drilldown },
-})
+let drilldown = ({ field, drilldown, additionalFields }, schema) =>
+  // value is a pipe-delimited list of values when drilldown is using additional fields (multi-term aggregation)
+  _.flow(_.split('|'), ([fieldValue, ...additionalFieldValues]) => [
+    {
+      term: { [getField(schema, field)]: fieldValue },
```

Comment:
I think we would want to check if there are additionalFields and either just use a single term, or multi_term like the buildGroupQuery below.

Reply:
> I think we would want to check if there are additionalFields and either just use a single term, or multi_term like the buildGroupQuery below. Unless I'm misunderstanding how drilldowns are handled.

This part is responsible for generating the filter of the `multi-term` key so if the node is doing a multi terms aggregation and generates a key field such as `Miami|FL` and we do a drilldown on it we are requiring the filter to filter both term `Miami` and `FL`. It is not related to how we group/bucket records (which is what I think you are referring to in the `buildGroupQuery` stage). I could be missing or misunderstanding something though.

Reply:
Yeah that makes sense 👍